src/main/java/ro/utcn/sd/alexh/assignment1/entity/Answer.java

```diff
@@ -20,6 +17,9 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Integer answerId;
     private Integer userId;
+    @Column(name = "question_id")
+    private Integer questionId;
```

Comment:
Would have been better to have a ManyToOne relationship towards the Question entity instead.

Reply:
Yes, but that would have required some refactoring. I didn't know exactly how Hibernate was supposed to work when I started the project. I'll keep that in mind, thank you!